public/locales/en/validation.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Please enter the scholarship name",
+  "deadline": "Please enter the scholarship deadline",
+  "validDate": "Please enter a valid date",
```

Comment:
Two things:
1. I suggest naming these `fieldPurpose`. E.g. `nameRequired`, `deadlineRequired`, `dateInvalid`, etc. 
2. I suggest sorting these. I think once you make the changes in the `ValidationSchema.ts` file you should be able to simply run `yarn i18next` and it'll find these keys and sort these files for you.